### PR TITLE
More descriptive Reset Settings warning

### PR DESCRIPTION
**Pull request by VitroidFPV**
Recently noticed an uptick in beginners mistaking the `Reset Settings` for a general "reset all the things I changed" or a "factory reset" button, leading to a lot of mistaken config wipes and the inability to connect to the FC afterward without a flash. This PR aims to make the result of clicking it clearer (and a red button to mark "danger")

New warning text in EN:
image

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Review by McGiverGim — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Pull request merged**
